README.md

```diff
@@ -114,14 +113,14 @@ need to be provided in order for this action to successfully run using one of th
 
 | Argument name             | Required | Description |
 | ------------------------- | -------- | ----------- |
-| `acrName`                 | Yes (for this scenario) | The name of the Azure Container Registry that the runnable application image will be pushed to. |
+| `acrName` or `registryUrl`| Yes (for this scenario)| The name of the Azure Container Registry or the base URL of any other container registry that the runnable application image will be pushed to. |
 | `appSourcePath`           | Yes (for this scenario) | Absolute path on the GitHub runner of the source application code to be built. |
 
 ### Arguments required for using an already pushed application image
 
 | Argument name             | Required | Description |
 | ------------------------- | -------- | ----------- |
-| `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. |
+| `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. If the image is found in any other container registry that requires authentication to pull, the `registryUrl` argument, or the `registryUsername` and `registryPassword` arguments, can be provided to authenticate requests to the container registry. |

```

Comment:
We should update the latter part of this sentence to say the following:

_"If the image is found in an other container registry that requires authentication to pull, the `registryUrl`, `registryUsername` and `registryPassword` arguments can be provided to authenticate requests to the container registry."_

With ACR, if the user doesn't provide `acrUsername` and `acrPassword`, we are still able to authenticate the call to ACR by generating an access token for the user, but we wouldn't be able to do that with a generic container registry, so we need them to provide all three values.

Reply:
Makes sense. Updated.